04_setup_ironic.sh

```diff
@@ -19,6 +19,7 @@ rm -f assets/templates/99_local-registry.yaml $OPENSHIFT_INSTALL_PATH/data/data/
 
 # Various commands here need the Pull Secret in a file
 export REGISTRY_AUTH_FILE=$(mktemp --tmpdir "pullsecret--XXXXXXXXXX")
+build_pull_secret
```

Comment:
Why not pass the filename here and have it write directly to a file instead of going via an environment variable?
Or even create the tempfile inside build_pull_secret and have that function return the filename.

Reply:
Good idea. I added another patch to the PR to do that.